File: PHP/Driver.php
<?php

class Driver extends Account {
    public $id;
    public $email;
    public $password;

    public function __construct($name, $document, $id, $email, $password) {
        parent::__construct($name, $document);
        $this->id = $id;
        $this->email = $email;
        $this->password = $password;
    }
}

?>

File: Java/Driver.java
class Driver extends Account {
    // Attributes
    Integer id;
    String email;
    String password;

    // Constructor
    public Driver(String name, String document, Integer id, String email, String password){
        super(name, document);
        this.id = id;
        this.email = email;
        this.password = password;
    }
}